client/src/components/ReviewEntry.jsx

```diff
@@ -8,6 +8,12 @@ const ReviewEntry = (props) => {
   let options = { month: 'long', day: 'numeric' };
   let postedDate = date.toLocaleString('en-US', options);
 
+  const handleContent = (e) => {
+    const element = document.getElementById('ReviewContent');
+    element.classList.remove('partial');
+    element.classList.add('expanded');
+  };
```

Comment:
Should we be using just React instead of mixing-and-matching React & native DOM methods?